packages/vue-i18n-core/test/legacy.test.ts

```diff
@@ -44,16 +44,28 @@ test('fallbackLocale', () => {
   expect(i18n.fallbackLocale).toEqual('ja')
 })
 
-test('availableLocales', () => {
-  const i18n = createVueI18n({
-    messages: {
-      en: {},
-      ja: {},
-      ru: {},
-      fr: {}
-    }
+describe('availableLocales', () => {
+  test('initialize messages', async () => {
+    const i18n = createVueI18n({
+      messages: {
+        en: {},
+        ja: {},
+        fr: {}
+      }
+    })
+    expect(i18n.availableLocales).toEqual(['en', 'ja', 'fr'].sort())
+  })
+  test('reactive update', () => {
```

Comment:
It is necessary to test with `watch` / `watchEffect` to see if the reaction is taking place.